core/src/utils/pid.cpp

```diff
@@ -1,19 +1,25 @@
 #include "../core/include/utils/pid.h"
 
 /**
-   * Create the PID object
-   */
+ * Create the PID object
+ */
 PID::PID(pid_config_t &config)
     : config(config)
 {
   pid_timer.reset();
 }
 
+void PID::init(double start_pt, double set_pt)
+{
+  set_target(set_pt);
+  reset();
+}
+
 /**
-   * Update the PID loop by taking the time difference from last update,
-   * and running the PID formula with the new sensor data
-   */
-void PID::update(double sensor_val)
+ * Update the PID loop by taking the time difference from last update,
+ * and running the PID formula with the new sensor data
+ */
```

Comment:
can we get a comment for what value this is returning?

Reply:
👍